tests/test_int_generator.py

```diff
@@ -0,0 +1,28 @@
+import os
+import unittest
+import pandas as pd
+import numpy as np
+
+from synthetic_data.distinct_generators.int_generator import random_integers
+
+class TestIntGenerator(unittest.TestCase):
+    def setUp(self):
+        self.rng = np.random.default_rng(12345)     
+
+    def test_return_type(self):
+        result = random_integers(self.rng)
+        self.assertIsInstance(result, np.ndarray)
+        for num in result:
+            self.assertIsInstance(num, np.int64)
+
+    def test_size(self):
+        num_rows = 5
```

Comment:
we should test more than 1 size to validate alterable and base case where we don't specify a size.